release_notes/374-release-notes.adoc

```diff
@@ -390,6 +390,15 @@ Release date: 5 June 2023
 ** link:https://nvd.nist.gov/vuln/detail/cve-2023-24539[CVE-2023-24539]
 ** link:https://nvd.nist.gov/vuln/detail/cve-2023-29400[CVE-2023-29400]
 
+[id="resolved-in-version-3745"]
+=== Resolved in version 3.74.5
+
+Release date: 13 July 2023
+
+* Fixed an issue with Sensor upgrade errors caused by PodSecurityPolicies that are not supported in EKS 1.25.
+
+* CVE-2023-<?>: Fixed by building Collector using an UBI image.
```

Comment:
CVE number?

Reply:
UBI = universal base image, but not sure we need to go into that level of detail. I have the same bug for 4.0.3 and have proposed this language:

Provides a Python3 security update. [(RHSA-2023:3591)](https://access.redhat.com/errata/RHSA-2023:3591)